introduction.md

```diff
@@ -0,0 +1 @@
+To finish the challeng on Code School Mastering Github 1.4
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Got it